budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/web/components/burntable/BurnTableWithFilter.html

```diff
@@ -1,8 +1,18 @@
 <wicket:panel>
 
     <div class="row">
-        <div wicket:id="filter"></div>
+        <div class="col-xs-10">
+            <div wicket:id="filter"></div>
+        </div>
+
+        <div class="col-xs-2">
+            <div class="form-group">
+                <label>Export</label><br />
+                <button class="btn btn-default btn-block" wicket:id="export" type="submit">Export to CSV</button>
```

Comment:
@daklassen Maybe we should position the button below the table. Otherwise it might confuse users that its located in the filter row.